corehq/mobile_flags.py

```diff
@@ -0,0 +1,14 @@
+from collections import namedtuple
+
+
+TAG_DIMAGI_ONLY = 'Dimagi Only'
```

Comment:
minor: it doesn't look like this is being used yet, would it be difficult to add in later if we end up extending this code and needing it?